src/swarm-scenario/Swarm/Game/World/Render.hs

```diff
@@ -38,8 +40,34 @@ import Swarm.Language.Pretty (prettyString)
 import Swarm.Util (surfaceEmpty)
 import Swarm.Util.Content
 import Swarm.Util.Erasable (erasableToMaybe)
+import Swarm.Util.Yaml
 import System.IO (hPutStrLn, stderr)
 
+newtype OneBitColor = OneBitColor Bool
+  deriving (Eq, Ord, Show, Generic, FromJSON, ToJSON)
+
+instance ToPixel OneBitColor where
+  toPixel (OneBitColor b) = case b of
+    False -> PixelRGBA8 0 0 0 255
+    True -> PixelRGBA8 255 255 255 255
+
+data ColorableCell
+  = OneBit OneBitColor
+  | Hex HexColor
+
+instance ToPixel ColorableCell where
+  toPixel (OneBit x) = toPixel x
+  toPixel (Hex x) = toPixel x
```

Comment:
@byorgey it seems this instance should be derivable.  Do you know how?

Reply:
Hmm, no, not any simple way at least.  I can imagine very overcomplicated things involving `Generic` but definitely not worth it unless we had many of these instances.

Reply:
@kostmo I suppose you could use [`DerivingVia`](https://kowainik.github.io/posts/deriving#via) if you have another similarly shaped instance around. 🤔

Reply:
"Similarly shaped" isn't enough, they have to be `Coercible`.